File: language/benchmarks/algorithms/filter-even/filter-even.py
def is_even(x: int) -> bool:
    return x % 2 == 0

def filter_even(xs: list[int]) -> list[int]:
    return list(filter(is_even, xs))

def main() -> list[int]:
    return filter_even([1, 2, 3, 4, 5, 6])

File: language/benchmarks/tokens/algorithms/filter-even/filter-even.ts
function isEven(x: number): boolean {
  return x % 2 === 0;
}

function filterEven(xs: number[]): number[] {
  return xs.filter(isEven);
}

function main(): number[] {
  return filterEven([1, 2, 3, 4, 5, 6]);
}
